types/agent.go

```diff
@@ -23,6 +23,7 @@ type Agent struct {
 	MetricsCount        uint64           `json:"metricsCount" yaml:"metricsCount"`
 	CreatedAt           time.Time        `json:"createdAt" yaml:"createdAt"`
 	UpdatedAt           time.Time        `json:"updatedAt" yaml:"updatedAt"`
+	Tags                []string         `json:"tags" yaml:"tags"`
```

Comment:
Can you move this up? Before the timestamps.

We are somehow ordering them by ID, required fields, optional fields, timestamps, optional relationships.